src/CreateEntityMappingView/CreateEntityMappingView.js

```diff
@@ -13,74 +13,129 @@ import ErrorMessage from './../Utils/ErrorMessage.js';
 
 import config from "../config/config";
 import AppStyles from "../AppStyles";
+import LoadingAnimation from "../Utils/LoadingAnimation";
 
 class CreateEntityMappingView extends ConnectionComponent {
 	constructor(props) {
 		super(props);
 		this.state = {
 			targetStatus: "",
-			selectedEventTypeId: {value: null, errorMessage: ""},
-			selectedEntityTypeId: {value: null, errorMessage: ""},
+			selectedEventType: {value: null, errorMessage: ""},
+			selectedEntityType: {value: null, errorMessage: ""},
 			mappings: CreateEntityMappingView.getDefaultMappings()
 		};
+		this.isCreateView = typeof this.props.match.params.entityMappingId === 'undefined';
```

Comment:
We cannot use props, as the view is accessed via url from the settings view and cannot be passed properties. We also can't pass this from the settings view, as we cannot read the entityMappingId from the URL when rendering a component in a route.